catalog/dags/providers/provider_api_scripts/justtakeitfree.py

```diff
@@ -0,0 +1,93 @@
+"""
+Content Provider:       Justtakeitfree
+
+ETL Process:            Use the API to identify all CC licensed media.
+
+Output:                 TSV file containing the media and the
+                        respective meta-data.
+
+Notes:                  https://justtakeitfree.com/api/api.php
```

Comment:
Can we leave a note that it needs authentication? Just linking to the GH issue would help as well.

Reply:
I looked at the other DAG notes, and we usually don't say whether the provider needs authentication or not. Do you think we should add it for every provider?

Reply:
Yes, I believe that would be helpful to know when running, especially because this comment gets displayed in the Airflow UI as well!